qiskit_experiments/randomized_benchmarking/interleaved_rb_analysis.py

```diff
@@ -50,37 +50,29 @@ def _run_analysis(
         plot: bool = True,
         ax: Optional["plotting.pyplot.AxesSubplot"] = None,
     ):
+
+        data = experiment_data.data()
+        num_qubits = len(data[0]["metadata"]["qubits"])
+
+        # Process data
         def data_processor(datum):
-            return level2_probability(datum, datum["metadata"]["ylabel"])
```

Comment:
I think having outcome label in circuit metadata is reasonable. I assumed processor option can be obtained from the metadata without extra data processing. Do you prefer having such extra data processing capability to generate data processor options?

https://github.com/Qiskit/qiskit-experiments/blob/1f64b4236864d28a7682da44e72f2bab2242013e/qiskit_experiments/analysis/curve_analysis.py#L367-L374

Reply:
I prefer having it as part of the metadata as well, but have no strong feelings about it (note that if we change this we can remove this metadata from `rb_experiment`).

Reply:
It seemed redundant to add the measurement label in metadata for every circuit when it's always the same and only depends on the number of qubits. Maybe it needs to be added if this is later a subclassed from a generic curve fit experiment.

Reply:
So we naively assume all `1` probability here (if `outcome` is not specified in the metadata)? I'm fine with this approach.